### PR TITLE
Windows compatibility.

### PR DESCRIPTION
**Pull request by ryzhyk**
Windows support with limited amount of testing.

@Kixiron

### CODE REVIEW

File: tools/install-flatbuf.sh [hunk 1]
```diff
@@ -7,20 +7,34 @@ retry() {
...
-    make
-    cd ..
+    if ( [ "$(uname)" = "Darwin" ] || [ "$(uname)" = "Linux" ] ); then
+        retry fetch_flatbuf_unix
+        cd flatbuffers
+        cmake -G "Unix Makefiles"
+        make
+        cd ..
+    else
+        retry fetch_flatbuf_unix
```
Comment (mbudiu-vmw) [inline tools/install-flatbuf.sh:29]: is this a bug?

File: tools/install-flatbuf.sh [hunk 2]
```diff
@@ -7,20 +7,34 @@ retry() {
...
 
-fetch_flatbuf() {
+fetch_flatbuf_unix() {
     rm -rf flatbuffers
     mkdir flatbuffers
     curl -L https://github.com/google/flatbuffers/archive/v1.11.0.tar.gz | tar -zx -C flatbuffers --strip-components 1
 }
 
+fetch_flatbuf_windows() {
+    curl -L https://github.com/google/flatbuffers/releases/download/v1.11.0/flatc_windows_exe.zip > fb.zip && unzip fb.zip
```
Comment (mbudiu-vmw) [inline tools/install-flatbuf.sh:17]: windows comes with curl?

**Review by mbudiu-vmw — APPROVED** [review]

File: tools/install-flatbuf.sh (hunk 1 above)
Comment (ryzhyk) [reply tools/install-flatbuf.sh:29]: Just bad naming.

File: tools/install-flatbuf.sh (hunk 2 above)
Comment (ryzhyk) [reply tools/install-flatbuf.sh:17]: Travis runs CI script in an environment that does. This is probably not going to work on all flavors of windows.

**Comment by Kixiron** [comment]
It builds and runs, thank you!

**Pull request merged**